dangogh/trafficcontrol
# WIP: Add Bindv9 DNS server to assist traffic router

#### What does this PR do?

Adds Bindv9 Container to CDN-In-A-Box
Adds static IPV4/IPV6 addresses to all containers
Zone file for *.cdn.local
Delegated sub-domain DNS for ciab.cdn.local --> trafficrouter.cdn.local

#### Which TC components are affected by this PR?

- [x] Other __CDN-In-A-Box_______

#### What is the best way to verify this PR?

1. Verify all containers start with static ipv4/ipv6 addresses
2. Verify that the cdn.local SOA can be retrieved from bind-utils.  The serial number is 1

$ host -t soa cdn.local 172.13.239.254
cdn.local has SOA record dns.cdn.local. hostmaster.cdn.local. 1 7200 3600 604800 38400

3. Update /etc/resolv.conf with DNS server IP 172.13.239.254 and test DNS resolution inside and outside the cdn.local authoritative zone.

$ host trafficrouter.cdn.local
$ host -6 trafficrouter.cdn.local
$ host www.google.com
$ host -6 www.google.com

4. Test trafficrouter DNS delegation (requires traffic router be running with correct CrConfig) 

$ host -t soa ciab.cdn.local 172.13.239.254

#### Check all that apply
